components/profile/components/EditProfile/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import { Row, Col, Input, Upload, Icon, Button, Modal, Typography } from 'antd';
+import * as constants from '../../constants';
+import "./index.scss";
+
+const { Title } = Typography;
+const { TextArea } = Input;
+const {
+    BUTTON_PRIMARY,
+    CLASS_NAMES,
+    EDIT_PROFILE_FIELDS,
+    EDIT_PROFILE_TITLE,
+    EDIT_PROFILE_UPLOAD_LIST_TYPE,
+    ICONS,
+} = constants;
+
+const {
+    antUploadText,
+    card,
+    infoField,
+    locationInput,
+    mb1,
+    mb2,
+    w100,
+    w80,
+} = CLASS_NAMES;
+
+const { plus } = ICONS;
+
+const {
+    bioLabel,
+    firstNameLabel,
+    getLocationLabel,
+    highSchoolLabel,
+    intendedUniLabel,
+    lastNameLabel,
+    locationLabel,
+    profilePhotoLabel,
+    profilePhotoPreviewAlt,
+    uploadLabel,
+} = EDIT_PROFILE_FIELDS;
+
+class EditProfile extends Component {
+    state = { 
+        fileList: [], location: null, locationLoading: false, 
+        previewImage: '', previewVisible: false, 
```

Comment:
if keys can not all fit on one line, put them one there own line

Reply:
okay. Noted